Training_analysis.ipynb

```diff
@@ -186,6 +191,28 @@
     "df = log.dataframe()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#login to AWS\n",
+    "# Uncomment and use this section of code if the machine you're using for analysis isn't already authenticated to your AWS Account\n",
+    "import os\n",
+    "#os.environ[\"AWS_DEFAULT_REGION\"] = \"\" #<-Add your region\n",
+    "#os.environ[\"AWS_ACCESS_KEY_ID\"] = \"\" #<-Add your access key\n",
+    "#os.environ[\"AWS_SECRET_ACCESS_KEY\"] = \"\" #<-Add you secret access key\n",
+    "#os.environ[\"AWS_SESSION_TOKEN\"] = \"\" #<-Add your session key if you have one\n",
+    "# Initialise S3 and load logs\n",
+    "import boto3\n",
+    "client = boto3.client('s3')\n",
+    "fh = S3FileHandler(bucket=\"\",prefix=\"\") #<-Add your S3 details, prefix should not including a'/' at the start or the end\n",
```

Comment:
Suggest to have variables that can be defined on a separate line. See Metrics.ipynb.

Reply:
made BUCKET and PREFIX variables on separate lines in line with Metrics.ipynb